perception/lidar_centerpoint/launch/lidar_centerpoint.launch.xml

```diff
@@ -19,37 +17,23 @@
       <composable_node pkg="lidar_centerpoint" plugin="centerpoint::LidarCenterPointNode" name="lidar_centerpoint">
         <remap from="~/input/pointcloud" to="$(var input/pointcloud)"/>
         <remap from="~/output/objects" to="$(var output/objects)"/>
-        <param name="score_threshold" value="$(var score_threshold)"/>
-        <param name="densification_world_frame_id" value="map"/>
-        <param name="densification_num_past_frames" value="1"/>
-        <param name="trt_precision" value="fp16"/>
-        <param name="has_twist" value="$(var has_twist)"/>
-        <param name="encoder_onnx_path" value="$(var model_path)/pts_voxel_encoder_$(var model_name).onnx"/>
-        <param name="encoder_engine_path" value="$(var model_path)/pts_voxel_encoder_$(var model_name).engine"/>
-        <param name="head_onnx_path" value="$(var model_path)/pts_backbone_neck_head_$(var model_name).onnx"/>
-        <param name="head_engine_path" value="$(var model_path)/pts_backbone_neck_head_$(var model_name).engine"/>
-        <param name="build_only" value="$(var build_only)"/>
-        <param from="$(var model_param_path)"/>
+        <param from="$(var model_param_path)" allow_substs="true"/>
         <param from="$(var class_remapper_param_path)"/>
+
+        <!-- This parameter shall NOT be included in param file. See the PR from the git blame for details. -->
+        <param name="build_only" value="$(var build_only)"/>
       </composable_node>
     </load_composable_node>
   </group>
   <group unless="$(var use_pointcloud_container)">
     <node pkg="lidar_centerpoint" exec="lidar_centerpoint_node" name="lidar_centerpoint" output="screen">
       <remap from="~/input/pointcloud" to="$(var input/pointcloud)"/>
       <remap from="~/output/objects" to="$(var output/objects)"/>
-      <param name="score_threshold" value="$(var score_threshold)"/>
-      <param name="densification_world_frame_id" value="map"/>
-      <param name="densification_num_past_frames" value="1"/>
-      <param name="trt_precision" value="fp16"/>
-      <param name="has_twist" value="$(var has_twist)"/>
-      <param name="encoder_onnx_path" value="$(var model_path)/pts_voxel_encoder_$(var model_name).onnx"/>
-      <param name="encoder_engine_path" value="$(var model_path)/pts_voxel_encoder_$(var model_name).engine"/>
-      <param name="head_onnx_path" value="$(var model_path)/pts_backbone_neck_head_$(var model_name).onnx"/>
-      <param name="head_engine_path" value="$(var model_path)/pts_backbone_neck_head_$(var model_name).engine"/>
-      <param name="build_only" value="$(var build_only)"/>
-      <param from="$(var model_param_path)"/>
+      <param from="$(var model_param_path)" allow_substs="true"/>
       <param from="$(var class_remapper_param_path)"/>
+
+      <!-- This parameter shall NOT be included in param file. See the PR from the git blame for details. -->
```

Comment:
nit: in my experience is customary to paste the link for the PR. this avoids cases where lints or similar change the blame content

Reply:
Fixed: https://github.com/autowarefoundation/autoware.universe/pull/6167/commits/118728088dcb810e6ced9d9c896dcbf4af409d6e